File: src/test/java/no/nav/familie/dokgen/DokgenTests.java
package no.nav.familie.dokgen;

import no.nav.familie.dokgen.Dokgen;
import org.junit.Test;
import org.junit.runner.RunWith;
import org.springframework.boot.test.context.SpringBootTest;
import org.springframework.test.context.ActiveProfiles;
import org.springframework.test.context.junit4.SpringRunner;


@RunWith(SpringRunner.class)
@SpringBootTest(classes = Dokgen.class)
@ActiveProfiles("dev")
public class DokgenTests {

    @Test
    public void contextLoads() {
    }

}

File: src/test/kotlin/no/nav/dokgen/DokgenTests.kt
package no.nav.dokgen


import org.junit.jupiter.api.Test
import org.junit.jupiter.api.extension.ExtendWith
import org.springframework.boot.test.context.SpringBootTest
import org.springframework.test.context.ActiveProfiles
import org.springframework.test.context.junit.jupiter.SpringExtension

@ExtendWith(SpringExtension::class)
@SpringBootTest(classes = [DokgenApplication::class])
@ActiveProfiles("dev")
class DokgenTests {
    @Test
    fun contextLoads() {
    }
}